clients/fides-js/src/lib/initialize.ts

```diff
@@ -264,13 +272,35 @@ export const initialize = async ({
       isPrivacyExperience(effectiveExperience) &&
       experienceIsValid(effectiveExperience, options)
     ) {
-      const updatedCookie = await updateCookie(
-        cookie,
-        effectiveExperience,
-        options.debug
-      );
-      Object.assign(cookie, updatedCookie);
-
+      if (options.fidesString) {
+        if (fetchedClientSideExperience) {
+          // if tc str was explicitly passed in, we need to override the client-side-fetched experience with consent from the cookie
+          // we don't update cookie because it already has been overridden by the injected fidesString
+          debugLog(
+            options.debug,
+            "Overriding preferences from client-side fetched experience with cookie fides_string consent",
+            cookie.fides_string
+          );
+          const tcfEntities = buildTcfEntitiesFromCookie(
+            effectiveExperience,
+            cookie
+          );
+          Object.assign(effectiveExperience, tcfEntities);
+        }
+      } else {
+        const updatedCookie = await updateCookie(
+          cookie,
+          effectiveExperience,
+          options.debug,
+          Boolean(options.fidesString)
```

Comment:
this line isn't needed anymore is it? `updateCookie` can always run, no need to check if there's an override prop? as in, can we remove the override prop?